### PR TITLE
Fix some issues with templates

### PR DESCRIPTION
**Pull request by neriyaco**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -11,11 +11,16 @@
             "python": "${workspaceFolder}/lib/python/python.exe",
             "program": "${workspaceFolder}/core/main.py",
             "console": "integratedTerminal",
-            "cwd": "D:\\tests\\sample_site",
+            "cwd": "C:\\Users\\ee102\\Documents\\ez-web-test-site",
```
Comment (binyamin555) [inline .vscode/launch.json:14]: No.

File: modules/data/database_repository.py [hunk 2]
```diff
@@ -41,6 +41,9 @@ def on_disconnect(self): ...
     def all(self, limit: int = None, skip: int = None):
         return self._session.query(self._model).limit(limit).offset(skip).all()
 
+    def query(self):
+        return self._session.query(self._model)
+
```
Comment (binyamin555) [inline modules/data/database_repository.py:46]: This is too low-level.

File: modules/pages/__main__.py [hunk 3]
```diff
@@ -4,55 +4,29 @@
...
-                raise ValueError(f"Invalid parameter kind for page: {param.kind}")
-        else:
-            parameters.append(param)
-
-    new_signature = signature.replace(parameters=parameters)
-
-    def page_route(*args, **kwargs):
-        return template.render(*args, **inject_kwargs, **kwargs)
-    
-    page_route.__signature__ = new_signature
```
Comment (binyamin555) [inline modules/pages/__main__.py]: Why did you remove this?

**Review by binyamin555 — CHANGES_REQUESTED** [review]
Maybe it's fine to directly use SQL alchemy directly in v1

File: modules/pages/__main__.py (hunk 3 above)
Comment (neriyaco) [reply modules/pages/__main__.py]: I made an oopsie.
I’ll fix it when I get home.

File: .vscode/launch.json (hunk 1 above)
Comment (neriyaco) [reply .vscode/launch.json:14]: This folder should really go in .gitignore

File: modules/data/database_repository.py (hunk 2 above)
Comment (neriyaco) [reply modules/data/database_repository.py:46]: So we need to discuss a different way of doing this. This functionality is pretty important.

File: .vscode/launch.json (hunk 1 above)
Comment (binyamin555) [reply .vscode/launch.json:14]: Then maybe you can do it here.

Comment (binyamin555) [reply .vscode/launch.json:14]: Then maybe you can do it here.

Comment (neriyaco) [reply .vscode/launch.json:14]: When I’ll get home/be in front of a computer.

**Review by binyamin555 — CHANGES_REQUESTED** [review]
Please add .vscode to .gitignore before merging.

**Comment by neriyaco** [comment]
> Please add .vscode to .gitignore before merging.

Sure, don’t approve yet.

**Review by binyamin555 — APPROVED** [review]

File: modules/data/database_repository.py (hunk 2 above)
Comment (binyamin555) [reply modules/data/database_repository.py:46]: Moved to #42
